pkgs/servers/nextcloud/packages/27.json

```diff
@@ -150,8 +150,8 @@
     ]
   },
   "maps": {
-    "sha256": "1gyxg5xp4mpdrw8630nqcf5yk8cs7a0kvfik2q01p05d533phc4d",
-    "url": "https://github.com/nextcloud/maps/releases/download/v1.2.0/maps-1.2.0.tar.gz",
+    "sha256": "049hrp79fj1bp9nk9isjrk427k238974x7gsj68jplxfrgq3sdkz",
+    "url": "https://github.com/nextcloud/maps/releases/download/v1.2.0-2-nightly/maps-1.2.0-2-nightly.tar.gz",
```

Comment:
@SuperSandro2000 Please fix.

Reply:
Actually I included the fix in https://github.com/NixOS/nixpkgs/pull/300126.

Reply:
We should fix the update script to stop doing that.